projects/plugins/jetpack/class.jetpack-admin.php

```diff
@@ -64,6 +64,7 @@ private function __construct() {
 
 		add_action( 'admin_init', array( $jetpack_react, 'react_redirects' ), 0 );
 		add_action( 'admin_menu', array( $jetpack_react, 'add_actions' ), 998 );
+		add_action( 'admin_menu', array( $jetpack_react, 'remove_jetpack_menu' ), 2000 );
```

Comment:
This line and the relevant function borrowed from #38768.
The priority has been changed to 2000 though as the Jetpack menu item was still appearing on JN test sites.